tests/index/test_index_document_meilisearch.py

```diff
@@ -197,6 +197,20 @@ def test_add_documents_csv(empty_index, songs_csv):
     assert index.get_primary_key() == "id"
 
 
+def test_add_documents_csv_with_delimiter(empty_index, songs_csv_custom_separator):
+    """Tests adding new documents to a clean index."""
+    index = empty_index("csv-delimiter")
+    response = index.add_documents_csv(songs_csv_custom_separator, csv_delimiter=";")
+    assert isinstance(response, TaskInfo)
+    assert response.task_uid is not None
+    task = index.wait_for_task(response.task_uid)
+    assert task.status == "succeeded"
```

Comment:
Also the task status, if the wait did not work your test will fail anyway right?

Reply:
It's possible for the task to complete, but the status to be failed right? I think this is what she is testing, that it was successful?

Reply:
When you read this:

```python
def test_add_documents_csv_with_delimiter(empty_index, songs_csv_custom_separator):
    """Tests adding new documents to a clean index."""

    index = empty_index("csv-delimiter")
    response = index.add_documents_csv(songs_csv_custom_separator, csv_delimiter=";")
    task = index.wait_for_task(response.task_uid)
    assert task.details["receivedDocuments"] == 53

    documents = index.get_documents().results
    assert documents[1].country == "Europe"
    assert documents[4].artist == "Elton John"
```

Could you let me know if you missed the removed assertions?